common/scripts/configs/master_integration_fields.sql

```diff
@@ -1061,6 +1062,42 @@ do $$
       values (269, '569cd11a1895ca4474700f95', 'certificates', 'string', false, true,'54188262bc4d591ba438d62a', '54188262bc4d591ba438d62a', '2017-11-03', '2017-11-03');
     end if;
 
+    -- masterIntegrationFields for kubernetes
+    if not exists (select 1 from "masterIntegrationFields" where "id" = 270) then
+      insert into "masterIntegrationFields" ("id", "masterIntegrationId", "name", "dataType", "isRequired", "isSecure","createdBy", "updatedBy", "createdAt", "updatedAt")
+      values (270, '59e71aa80552d20500e76bb0', 'clusterAccessType', 'string', true, false,'54188262bc4d591ba438d62a', '54188262bc4d591ba438d62a', '2016-06-01', '2016-06-01');
```

Comment:
Space after `,`.

Reply:
Let's update the dates to today's. Also, make sure the isSecure matches whatever we had earlier.